### PR TITLE
Add  tcp and unix spapr-vty test cases on ppc

### PR DESCRIPTION
**Pull request by MiriamDeng**
Signed-off-by: Miriam Deng mdeng@redhat.com

### CODE REVIEW

**Comment by MiriamDeng** [comment]
ID:1322229

**Comment by MiriamDeng** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/502

**Comment by CongLi** [comment]
It's better to set those cmds (ls, pwd, reboot) and match strings as variants,
or there are lots of dup code in your case.

**Comment by MiriamDeng** [comment]
@coli,thank you.I will have a look.

**Comment by mazhang** [comment]
Maybe you could according the function virtio_serial_login() in virtio_port_login.py write this case.

**Comment by MiriamDeng** [comment]
Update code according to comments,thanks !

**Comment by MiriamDeng** [comment]
Logs
JOB LOG    : /root/test_home/mdeng/staf-kvm-devel/workspace/job-results/job-2016-11-01T03.58-d55af8e/job.log
TESTS      : 2
 (1/2) smp_2.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.ppc64le.io-github-autotest-qemu.spapr_vty.unix: PASS (65.20 s)
 (2/2) smp_2.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.ppc64le.io-github-autotest-qemu.spapr_vty.tcp: PASS (90.90 s)

File: qemu/tests/spapr_vty.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    def check_reboot_cmd(session, cmd):
+        error.context("Send a '%s' command to the guest" %
+                      cmd, logging.info)
+        session.cmd(cmd, timeout=1, ignore_all_errors=True)
+
+    try:
+        error.context("Get the main VM", logging.info)
+        vm = env.get_vm(params["main_vm"])
+        vm.verify_alive()
```
Comment (qunfang) [inline qemu/tests/spapr_vty.py]: The "try" part is so big, and if so even the vm does not boot up successfully, it falls into the "spapr-vty" failure.

**Comment by MiriamDeng** [comment]
Actually,some faults are executed in the function named by "verify_alive"
    def verify_alive(self):
        """
        Make sure the VM is alive and that the main monitor is responsive.

        :raise VMDeadError: If the VM is dead
        :raise: Various monitor exceptions if the monitor is unresponsive
        """
        self.verify_disk_image_bootable()
        self.verify_userspace_crash()
        self.verify_kernel_crash()
        self.verify_illegal_instruction()
        self.verify_kvm_internal_error()
        try:
            virt_vm.BaseVM.verify_alive(self)
            if self.monitor:
                self.monitor.verify_responsive()
        except virt_vm.VMDeadError:
            raise virt_vm.VMDeadError(self.process.get_status(),
                                      self.process.get_output())

**Comment by MiriamDeng** [comment]
In order to improve the code I add some exceptions for other steps,thanks !

**Pull request closed without merging**